deploy/olm-catalog/opendatahub/1.4.0/manifests/opendatahub-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,270 @@
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/cluster"
+                  }
+                },
+                "name": "prometheus-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/operator"
+                  }
+                },
+                "name": "prometheus-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "additional"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "jupyterhub/notebook-images"
+                  }
+                },
+                "name": "notebook-images"
+              }
+            ],
+            "repos": [
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/v1.4"
+              }
+            ]
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: AI/Machine Learning, Big Data
+    description: "Open Data Hub Operator for deployment and management of Open Data Hub and Kubeflow on OpenShift"
+    support: Open Data Hub
+    repository: https://github.com/opendatahub-io/opendatahub-operator
+    createdAt: '2022-10-18T00:00:00Z'
+    containerImage: quay.io/opendatahub/opendatahub-operator:v1.4.0
+    certified: 'False'
+  name: opendatahub-operator.v1.4.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - description: KfDef is the Schema for the applications API
+        kind: KfDef
+        name: kfdefs.kfdef.apps.kubeflow.org
+        version: v1
+        displayName: Open Data Hub
+        group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform. Open Data Hub integrates multiple AI/ML open source components into one operator that can easily be downloaded and installed by OpenShift users.
+
+    Open Data Hub operator allows users to install and manage components of the Open Data Hub. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the
+    projects share some components, but can be mostly seen as an extension of each other to provide a complete solution for both novice and skilled enterprise users.
+
+    ### Components
+    * Open Data Hub Dashboard v2.2.1  - A web dashboard that displays installed Open Data Hub components with easy access to component UIs and documentation
+    * ODH Notebook Controller - Secure management of Jupyter Notebook in Kubernetes environments built on top of Kubeflow Notebook Controller with support for OAuth
```

Comment:
@samuelvl Any changes you want to make to this line?

Reply:
No, same applies for 1.4